.ci/docker/common/install_android.sh

```diff
@@ -12,7 +12,7 @@ apt-get autoclean && apt-get clean
 rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 pushd /tmp
-curl -Os --retry 3 $_https_amazon_aws/android-ndk-${ANDROID_NDK}-linux-x86_64.zip
+curl -Os --retry 3 https://dl.google.com/android/repository/android-ndk-${ANDROID_NDK}-linux-x86_64.zip
```

Comment:
Please don't do that, as `dl.google.com` often rate-limits our CI. Let me upload NDK-19 to our mirror

Reply:
Done

Reply:
Fixed, but I need your help to upload.